flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/channel/ChannelStateWriterImplTest.java

```diff
@@ -135,145 +145,200 @@ public void testAbortOldAndStartNewCheckpoint() throws Exception {
                     worker.processAllRequests();
 
                     ChannelStateWriteResult result42 = writer.getAndRemoveWriteResult(checkpoint42);
-                    assertTrue(result42.isDone());
+                    assertThat(result42.isDone()).isTrue();
                     try {
                         result42.getInputChannelStateHandles().get();
                         fail("The result should have failed.");
                     } catch (Throwable throwable) {
-                        assertTrue(findThrowable(throwable, TestException.class).isPresent());
+                        assertThat(findThrowable(throwable, TestException.class).isPresent())
+                                .isTrue();
```

Comment:
```suggestion
                        assertThat(findThrowable(throwable, TestException.class)).isPresent();
```
assertJ supports some `Optional` methods

Reply:
Change to this:

```
                    assertThatThrownBy(() -> result42.getInputChannelStateHandles().get())
                            .as("The result should have failed.")
                            .hasCauseInstanceOf(TestException.class);
```